test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -2375,6 +2375,12 @@ public interface CheckedFunction
         void apply() throws Throwable;
     }
 
+    @FunctionalInterface
+    public interface CheckedSupplier
+    {
+        ResultMessage apply() throws Throwable;
```

Comment:
NIT: should we align with the JDKs `Supplier` API? In that case, this method would need to be `get` 
```suggestion
        ResultMessage get() throws Throwable;
```